roles/rook/README.rst

```diff
@@ -0,0 +1,106 @@
+Rook
+====
+
+This role is used for updating and recovering the rook jenkins in the rook ci vpc.
+
+Those are the functions in this role are
+
+- rook-jenkins-update For updating rook jenkins version to the version defined in the "jenkins_master_image" varible
+
+- rook-os-update For updating rook jenkins OS packages
+
+- rook-recovery For recoverying the Prod-jenkins instance from the image defined in the "image" varible in a case that the instance was deleted or crashed
+
+Usage
++++++
+
+The rook role is primarily used by the ``rook.yml`` playbook.  Run this playbook with one of the optional tags listed in the ``rook.yml``
+to upgrade rook jenkins os/app or recoverying the instance from latest image
+
+**NOTE:** ``rook.yml`` Is currently using only localhost and not any host from the inventory in the inventory this is because the rook-recovery creates and configures the rook jenkins during his run.
+
+**NOTE:** After running play with the rook-recovery tag you will need to make the new recoverd instance available the to public network once the instance is deployed, by adding it to the load balancing target group under the rook ci vpc access details to the aws dashboard can be found in here http://wiki.ceph.redhat.com/dokuwiki/doku.php?id=rook_aws_account
```

Comment:
1. s/rook-recovery/``rook-recovery``
2. s/recoverd/recovered
3. Split this into sentences.  Maybe "instance is deployed.  To do this, add it to the load balancing target group" ...